### DISCUSSION TITLE
Different beta values using "openSesame"

### DISCUSSION THREAD

**Discussion by yazdan59** [zwdzwd/sesame#181] [General]
Hello,
I got different beta values when I use the following two codes while I expect they are equivalent codes and they should create similar results. Do I miss something in understanding the concept?

sdfs_prep <- openSesame(sdfs, prep="QCDPB", func=NULL)
betas <- openSesame(sdfs_prep)
betas[1:3,1:3]
           _206112280175_R03C01 206112280082_R03C01 206112290047_R03C01
cg00000029           0.7992652           0.6897070           0.6676159
cg00000103                  NA                  NA                  NA
cg00000109           0.9728390           0.9733719           0.9772612_

sdfs_prep_2 <- openSesame(sdfs, prep="QCDPB", func=getBetas)
sdfs_prep_2[1:3,1:3]
           _206112280175_R03C01 206112280082_R03C01 206112290047_R03C01
cg00000029           0.7166326           0.6450280           0.6308799
cg00000103                  NA                  NA                  NA
cg00000109           0.9214796           0.9207276           0.9318405_

**Comment by zwdzwd** [comment]
Hi,
The top essentially did all the preprocessing again. That's why they are different. The second is essentially 
```
getBetas(openSesame(sdfs, prep="QCDPB", func=NULL))
```

**Reply by yazdan59** [reply]
Thanks for your explanation.
